chains/ethereum/chain.go

```diff
@@ -116,11 +117,21 @@ func InitializeChain(chainCfg *core.ChainConfig, logger log15.Logger, sysErr cha
 	if err != nil {
 		return nil, err
 	}
-	err = conn.EnsureHasBytecode(cfg.erc20HandlerContract)
+
+	if cfg.erc20HandlerContract == utils.ZeroAddress {
```

Comment:
do not this looks bulky for you? !=

Reply:
Yup definitely does, pushed an improvement